src/typescript-service.ts

```diff
@@ -270,8 +284,6 @@ export default class TypeScriptService {
                 const configuration = this.projectManager.getConfiguration(fileName);
                 configuration.prepare(this.connection).then(() => {
                     try {
-                        this.projectManager.syncConfiguration(configuration, this.connection); // resync after getting new config
```

Comment:
Isn't this necessary, because some files might have previously held the placeholder content?